src/ProjectTemplates/Web.ItemTemplates/content/RazorComponent/Index.razor

```diff
@@ -0,0 +1,5 @@
+<h3>Index</h3>
```

Comment:
It would be nice if this is a parameter (if possible)

Reply:
@danroth27

Reply:
If one of the razor people would like to inform me of some syntax like `@Component.Name` that will work here I'd be happy to use it.